### ISSUE TITLE
preprocessor conditional error in gstutils

### ISSUE THREAD

**Issue by artificiel** [openframeworks/openFrameworks#7846]
I think the side-effect is that v 1.0-1.18 end up using the second form while the first form gets executed twice for 0.n (the bad #elseif simply gets ignored), which might not be a true bug (and might not happen often) but the conditional should be fixed anyway (I don't know if it's simpler to blanket 1.n with the second form as it seems harmless, of if the conditional should really be operational).

https://github.com/openframeworks/openFrameworks/blob/aa774a774d69872baa3542402a57cdfc8a6eed51/libs/openFrameworks/video/ofGstUtils.cpp#L307

**Comment by ofTheo** [comment]
ahh thanks @artificiel so is the fix then just changing it to #elif ?

**Comment by artificiel** [comment]
yes that's the direct fix, but the WIP ambiance of the surrounding code (and the fact the error itself did not cause apparent trouble) made me wonder if a deeper look at the conditionals was warranted.
